src/CSF.SRDashboard.Client/Services/IWorkLoadManagementService.cs

```diff
@@ -0,0 +1,20 @@
+﻿using CSF.SRDashboard.Client.Components.Tables.WorkloadRequest.Entities;
+using CSF.SRDashboard.Client.DTO.WorkLoadManagement;
+using CSF.SRDashboard.Client.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading.Tasks;
+
+namespace CSF.SRDashboard.Client.Services
+{
+    public interface IWorkLoadManagementService
+    {
+        WorkItemDTO GetByWorkItemById(int id);
+        List<WorkItemDTO> GetByLineOfBusinessId(string lineOfBusinessId);
+        WorkItemDTO AddWorkItem(WorkItemDTO workItem);
+        List<WorkItemDTO> GetByCdnNumber(string cdn);
+        WorkItemDTO PostRequestModel(RequestModel requestModel, IGatewayService gatewayService);
+        List<WorkloadRequestTableItem> GetByCdnInRequestTableFormat(string cdn);
```

Comment:
We could consider adding documentation to these methods.
```suggestion
        WorkItemDTO GetByWorkItemById(int id);
        
        List<WorkItemDTO> GetByLineOfBusinessId(string lineOfBusinessId);
        
        WorkItemDTO AddWorkItem(WorkItemDTO workItem);
        
        List<WorkItemDTO> GetByCdnNumber(string cdn);
        
        WorkItemDTO PostRequestModel(RequestModel requestModel, IGatewayService gatewayService);
        
        List<WorkloadRequestTableItem> GetByCdnInRequestTableFormat(string cdn);
```

Reply:
Added spacing and also added documentation